src/Directory.Build.props

```diff
@@ -6,8 +6,8 @@
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <DisableImplicitFrameworkDefines>true</DisableImplicitFrameworkDefines>
     <DisableImplicitConfigurationDefines>true</DisableImplicitConfigurationDefines>
-
     <CodeAnalysisRuleset>$(MSBuildThisFileDirectory)CodeAnalysis.ruleset</CodeAnalysisRuleset>
+    <EnableXUnitAnalyzer>true</EnableXUnitAnalyzer>
```

Comment:
Is it a problem (noticable build time increase, etc) to have this for the src projects as well as test? (Should this line be in a different props file or a propertygroup... wherever we have stuff that is different for tests (collectively)?)

Reply:
No this property is simply ignored on source projects, we don't have a separate place for source vs test properties.